src/Client/Validation/HttpClientValidator.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Client\Validation;
+
+use K911\Swoole\Client\Http;
+use K911\Swoole\Client\Validation\Exception\ClientConnectionError;
+use K911\Swoole\Client\Validation\Exception\InvalidHttpMethod;
+use K911\Swoole\Client\Validation\Exception\MissingContentType;
+use Swoole\Coroutine\Http\Client;
+
+class HttpClientValidator
```

Comment:
Yeah, like I've noted earlier, for now we should drop this class and use private methods of internal HttpClient class (internal means that it can be used only in library source code, not by others using library).